README.md

```diff
@@ -39,23 +42,32 @@ dnf install mpv-devel
 
 [**Download latest release**](https://github.com/DucNg/fip-player/releases/download/latest/fip-player)
 
+Make the binary file executable:
+
 ```shell
 chmod +x fip-player
 ./fip-player
 ```
- 
-Install
-```
+
+Install system-wide:
+  
+```shell
 sudo cp fip-player /usr/local/bin
 ```
 
+Install for current user (`~/.local/bin/` must be in your PATH):
+
+```shell
+cp fip-player ~/.local/bin
+```
+
 ### Install using the golang toolchain
 
 ```shell
 go install github.com/DucNg/fip-player@latest
 ```
 
-## Build and run
+### Build and run
```

Comment:
Maybe add some instructions about the Makefile in this section now that it's available?

Reply:
well i have really no idea why is it a good idea to have a makefile in this project at all since we are basically using the go toolchain for everything

Reply:
Yes you're right :sweat_smile: 

I forgot that I added it into the "nice to have". I probably added it having in mind other projects with man pages. Let's leave it like this until it's actually useful for something I guess 🤷‍♂️